moped-editor/src/views/staff/StaffForm.js

```diff
@@ -97,9 +130,15 @@ const StaffForm = ({ editFormData = null }) => {
 
     // POST or PUT request to User Management API
     const requestString = editFormData === null ? "post" : "put";
-    const requestPath = "/users/";
+    const requestPath =
+      editFormData === null ? "/users/" : "/users/" + userCognitoId;
 
-    requestApi(requestString, requestPath, data);
+    // If editing and password is not updated, remove it
+    if (editFormData) {
+      delete data.password;
```

Comment:
this seems like it would work consistently but I wonder if there are any form state handlers that tell you if a certain part of the form state has been edited. In some libraries I've seen things like `isDirty`. Anyways, just food for thought. I'm enjoying learning about these new form library features.

Reply:
Ah, glad that you brought this up! That is exposed by React Hook Form, but I didn't think about it here. I'll add this update to make this less manual and get more familiar with how that works. 👍